src/libs/actions/Policy.js

```diff
@@ -136,8 +140,35 @@ function invite(login, welcomeNote, policyID) {
         });
 }
 
+/**
+ * Merges the passed in login into the specified policy
+ *
+ * @param {String} name
+ */
+function create(name) {
+    Policy_Create({type: 'free', policyName: name})
```

Comment:
We should use a `CONST` for this like `CONST.POLICY.TYPE`

Reply:
Done!